h5py/h5p.pyx

```diff
@@ -1154,6 +1182,7 @@ cdef class PropFAID(PropInstanceID):
         - h5fd.MPIO
         - h5fd.MULTI
         - h5fd.SEC2
+        - h5fd.DIRECT  (if available)
         - h5fd.STDIO
```

Comment:
I guess ROS3 should also be in this list, and it was just an oversight that it's not.

Reply:
I added it.